api/config/packages/doctrine_migrations.yaml

```diff
@@ -7,6 +7,6 @@ doctrine_migrations:
         # as migrations classes should NOT be autoloaded
         'DoctrineMigrations': '%kernel.project_dir%/migrations/schema'
         'DataMigrations': '%kernel.project_dir%/migrations/%data_migrations_dir%'
-    enable_profiler: '%kernel.debug%'
+    enable_profiler: false
```

Comment:
`doctrine_migrations:enable_profiler`
Didn't know what this is used for, but it seems there's a [tab for doctrine migrations](https://github.com/doctrine/DoctrineMigrationsBundle/pull/407) in the symfony debug panel. Never used myself, but it actually looks like this could be useful. So I'm ok to enable this again. In the api-platform upstream repo this was disabled in commit https://github.com/api-platform/api-platform/commit/6fd5050af32a50ad327aca543d3cd567b8959c63, but there's no further description, on why this was disabled.

`framework:profiler:collect_serializer_data`
This is related to a new functionality in Symfony 6.1 which [displays serializer info](https://symfony.com/blog/new-in-symfony-6-1-serializer-profiling) in the debug panel. Default value is `false`, upstream repo has changed to `true`. Having this enabled conflicts with our implementation of `TranslationConstraintViolationListNormalizer`. Before we can enable this, further investigation is needed to figure out what exactly breaks.